src/index.ts

```diff
@@ -2,7 +2,7 @@ import http from 'http'
 import app from './app'
 import { sequelize as dbConnection } from './db/connection'
 
-const port = 4000
+const port = 3000
```

Comment:
Why you changed the port? :eyes:, and Ahmed we have a port in the configs, so why we did not use it?